File: src/main/graphql/io/github/graphql/resolver/GistCommentResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for GistComment
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-05T19:09:00+0800"
)
interface GistCommentResolver {

    @Throws(Exception::class)
    fun userContentEdits(gistCommentTO: GistCommentTO, after: String?, before: String?, first: Int?, last: Int?): UserContentEditConnectionTO?

}

File: src/main/scala-graphql/io/github/graphql/scala/resolver/GistCommentResolver.scala
package io.github.graphql.scala.resolver

import io.github.graphql.scala.model._

/**
 * Resolver for GistComment
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
trait GistCommentResolver {

    @throws[Exception]
    def userContentEdits(gistCommentTO: GistCommentTO, after: String, before: String, first: Option[Int], last: Option[Int]): UserContentEditConnectionTO

}
